letsencrypt/cli.py

```diff
@@ -696,6 +696,15 @@ def parse_args(self):
         parsed_args = self.parser.parse_args(self.args)
         parsed_args.func = self.VERBS[self.verb]
 
+        # Do any post-parsing homework here
+
+        # argparse seemingly isn't flexible enough to give us this behaviour easily...
+        staging_uri = 'https://acme-staging.api.letsencrypt.org/directory'
```

Comment:
This should be moved to `constants.py` and all other references to the URL should be changed to use the constant. This URL could change and we don't want to create new bugs/problems/work for us in the future.